### ISSUE TITLE
Card box lid problem

### ISSUE THREAD

**Issue by myk3y** [florianfesti/boxes#209]
I could be doing something wrong, but the lid for every card box I've made ends up with tabs on one side, where it should be a plain edge on all four sides.

**Comment by florianfesti** [comment]
Yeah, the picture is a bit outdated. The tabs there are actually intentional. There is a narrow piece that fits in there that acts as a grip to be able to open the lid more easily.
Guess I finally need to add some assembly instructions as you are already the second person to get confused about this.

**Comment by myk3y** [comment]
Thanks Florian.

Makes sense now = doh! :)

**Comment by myk3y** [comment]
User error

**Comment by bwente** [comment]
I thought I had done something wrong also. I came back to double check the photo to see how to put it together. I got it figured out after reading these issues.

Here is my final assembly, maybe it will help the next person.

![IMG_5253](https://user-images.githubusercontent.com/371964/82721552-116e0980-9c8c-11ea-9029-b990c9417a47.jpg)

![IMG_5252](https://user-images.githubusercontent.com/371964/82721572-3d898a80-9c8c-11ea-95b3-4ed19e4bdbe4.jpg)

This was a gift for my brother-in-law.
